commercial measures/SWCR007-05 Float Temp Ctrl/SWCR007-05 Float Temp Ctrl_Ex/cases/Gro&0&cDXGF&Ex&Ref_Storage__RefControl.csv

```diff
@@ -0,0 +1,10 @@
+skip,case_name,:refsys_condenser_type,:refsys_cond_fan_speed_control_type,:refsys_cond_sct_delta,:refsys_cond_floating
+#,defaults,,,,
+,NE-Ref_Storage-RefControl-FltSucTemp-AirCool ,AirCooled,TwoSpeed,12,TRUE
+,NE-Ref_Storage-RefControl-FixSucTemp-AirCool ,AirCooled,TwoSpeed,12,TRUE
```

Comment:
Several changes needed
1. Rename cases from FltSucTemp/FixSucTemp to FltCondTemp/FixCondTemp
2. Adjust values in cond_sct_delta column from 12/17 {F} to 6.67/9.44 {C}.
3. Adjust values in cond_floating column from TRUE to FALSE where the case is FixCondTemp.

Reply:
Thanks for your comment. 
I forgot to change the temp from IP unit to SI unit. Now, revised.
The other 2 comments were addressed.